input/pagecontent/actors.md

```diff
@@ -2,11 +2,11 @@ The following ActorDefinitions define the systems that plays a role in data exch
 
 - [AU eRequesting Placer actor](ActorDefinition-au-erequesting-actor-placer.html)
 
-The AU eRequesting Placer is a system that is responsible for initiating and creating diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
+The AU eRequesting Placer is a system that is responsible for creating diagnostic requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
```

Comment:
I don't think this implements the [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.